From ec978d563c38734cb7d1fd33c1a9422308ba1b1b Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Fri, 24 Mar 2023 18:36:02 +0100
Subject: [PATCH 468/529] ARM: dts: stm32: enable i2c5 in
 stm32mp157x-dk2-a7-examples

Enable the i2c5 in dk2 a7 examples.

Change-Id: I70a249859a68b4ab942e1a7b527f7175f7820e49
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/297457
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Valentin CARON <valentin.caron@foss.st.com>
Reviewed-by: Alain VOLMAT <alain.volmat@foss.st.com>
---
 arch/arm/boot/dts/stm32mp157c-dk2-a7-examples.dts | 4 ++++
 arch/arm/boot/dts/stm32mp157f-dk2-a7-examples.dts | 4 ++++
 2 files changed, 8 insertions(+)

--- a/arch/arm/boot/dts/stm32mp157c-dk2-a7-examples.dts
+++ b/arch/arm/boot/dts/stm32mp157c-dk2-a7-examples.dts
@@ -58,6 +58,10 @@
 	};
 };
 
+&i2c5 {
+	status = "okay";
+};
+
 &timers1 {
 	status = "okay";
 };
--- a/arch/arm/boot/dts/stm32mp157f-dk2-a7-examples.dts
+++ b/arch/arm/boot/dts/stm32mp157f-dk2-a7-examples.dts
@@ -58,6 +58,10 @@
 	};
 };
 
+&i2c5 {
+	status = "okay";
+};
+
 &timers1 {
 	status = "okay";
 };
